### PR TITLE
Add nolus-protocol testnet

### PR DESCRIPTION
**Pull request by ivivanov**
## What is the purpose of the change

With this pull request we want to add nolus-protocol's native token NLS. Currently we have only testnet available. 

## Brief Changelog
- *Add nolus testnet chain info*
- *Add nolus token logo svg*
- *Add info for IBC channel between osmosis testnet and nolus testnet*

## Testing and Verifying
This change has been tested locally by rebuilding the website and verified content and links are expected

## Documentation and Release Note
  - Does this pull request introduce a new feature or user-facing behavior changes? *no*
  - How is the feature or change documented?  *not applicable*

### CODE REVIEW

File: packages/web/config/price.ts [hunk 1]
```diff
@@ -1355,8 +1355,18 @@ export const PoolPriceRoutes: IntermediateRoute[] = [
...
-        "cw20:juno1u8cr3hcjvfkzxcaacv9q75uw9hwjmn8pucc93pmy6yvkzz79kh3qncca8x"
+      [{ portId: "transfer", channelId: "channel-169" }],
+      "cw20:juno1u8cr3hcjvfkzxcaacv9q75uw9hwjmn8pucc93pmy6yvkzz79kh3qncca8x"
+    ),
+    spotPriceDestDenom: "uosmo",
+    destCoinId: "pool:uosmo",
+  },
+  {
+    alternativeCoinId: "pool:nls",
+    poolId: "781",
```
Comment (JeremyParish69) [inline packages/web/config/price.ts:1366]: This is an aUSDC pool, not an OSMO pool, so this pricing won't work.

File: packages/web/config/ibc-assets.ts [hunk 2]
```diff
@@ -139,6 +139,13 @@ export const IBCAssetInfos: (IBCAsset & {
           sourceChainTokens: [AxelarSourceChainConfigs.wftm.fantom],
         },
       },
+      {
+        counterpartyChainId: "nolus-rila",
+        sourceChannelId: "channel-1837",
+        destChannelId: "channel-0",
+        coinMinimalDenom: "unls",
+        isVerified: true,
```
Comment (JeremyParish69) [inline packages/web/config/ibc-assets.ts:147]: I assume we don't mind that this is set to true?

**Review by jonator — APPROVED** [review]

File: packages/web/config/chain-infos.ts [hunk 3]
```diff
@@ -2896,6 +2896,36 @@ const chainInfos = (
       features: ["ibc-transfer", "ibc-go"],
       explorerUrlToTx: "https://explorer.8ball.info/8ball/tx/{txHash}",
     },
+    {
+      rpc: IS_TESTNET ? "https://net-rila.nolus.io:26657" : "",
+      rest: IS_TESTNET ? "https://net-rila.nolus.io:1317" : "",
+      chainId: IS_TESTNET ? "nolus-rila" : "",
+      chainName: IS_TESTNET ? "Rila Testnet" : "",
```
[the remark this answers is not in the document]
Comment (ivivanov) [reply packages/web/config/chain-infos.ts:2903]: Thanks for the recommendation

File: packages/web/config/price.ts [hunk 4]
```diff
@@ -1355,10 +1355,20 @@ export const PoolPriceRoutes: IntermediateRoute[] = [
...
     destCoinId: "pool:uosmo",
   },
+  {
+    alternativeCoinId: "pool:unls",
+    poolId: "781",
+    spotPriceSourceDenom: DenomHelper.ibcDenom(
+      [{ portId: "transfer", channelId: "channel-1837" }],
+      "unls"
+    ),
+    spotPriceDestDenom: "aUSDC",
```
Comment (JeremyParish69) [inline packages/web/config/price.ts:1371]: Please use the denom helper function here:
```
[{ portId: "transfer", channelId: "channel-1946" }],
      "uausdc"
    )
```

File: packages/web/config/price.ts [hunk 5]
```diff
@@ -1355,10 +1355,20 @@ export const PoolPriceRoutes: IntermediateRoute[] = [
...
+    destCoinId: "pool:aUSDC",
```
Comment (JeremyParish69) [inline packages/web/config/price.ts:1372]: The coin id hasn't been defined, yet, so we could probably just use the coingecko price?
`destCoinId: "usd-coin"`

File: packages/web/config/price.ts (hunk 4 above)
Comment (ivivanov) [reply packages/web/config/price.ts]: Fixed

File: packages/web/config/price.ts (hunk 5 above)
Comment (ivivanov) [reply packages/web/config/price.ts]: Fixed

**Comment by ivivanov** [comment]
@JeremyParish69 both deposit and withdrawal initiated from the UI are working as expected:
deposit: https://explorer-rila.nolus.io/nolus-rila/tx/6E3F82D7C03053822062BE6EE1C9C7C097D335360F2AEAD91F0FA6FA11F86CC6
withdrawal: https://testnet.mintscan.io/osmosis-testnet/txs/2447E16C71AC9F2A736B59B2C38F616A53278BD1376CBA03DD09A7BE6A86CCE6

**Pull request merged**